README.adoc

```diff
@@ -1,34 +1,125 @@
-////
-This is the base template for Hazelcast integration module guides.
-
-You can clone this repository, work on it and create your guide, and then push to a new repository.
-////
-
-:github-address: [github-repo-address-of-your-guide]
+:github-address: https://github.com/hazelcast-guides/getting-started-hazelcast-springboot
```

Comment:
`getting-started-hazelcast-client-springboot`
`getting-started-hazelcast-embedded-springboot`
are better names. We can even drop `getting-started` as we will group those guides under a section named `Getting Started`

I expect more guides as below over time.
`hazelcast-client-quarkus`
`hazelcast-client-micronaut`
`hazelcast-embedded-micronaut`

Reply:
Similar one for getting started web list --> https://cloud.google.com/gcp/getting-started#quick-starts

Reply:
Done 👍